Dockerfile

```diff
@@ -43,9 +41,6 @@ RUN if [ x"$( sha256sum /home/lisk/wait-for-it.sh |awk '{print $1}' )" = x"${WFI
       exit 1; \
     fi
 
-ENV LISK_API_WHITELIST=127.0.0.1
```

Comment:
Why we don't need this anymore?

Reply:
It's in the defaults now: https://github.com/LiskHQ/lisk/blob/development/config/default/config.json#L35